packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -447,9 +443,21 @@ export const Audio = () => {
     }
 
     currentUriRef.current = newUri
-    const imageUrl = trackImageSource?.source?.[2]?.uri ?? DEFAULT_IMAGE_URL
+    const imageUrl =
+      getImageSourceOptimistic({
+        cid: track ? track.cover_art_sizes || track.cover_art : null,
+        user: trackOwner
+        // localSource
```

Comment:
comment

Reply:
Gonna leave this in as an indicator that we still need to make this work with localSource